ironfish-cli/src/commands/deposit.ts

```diff
@@ -112,20 +113,28 @@ export default class Bank extends IronfishCommand {
 
     const balanceResp = await this.client.getAccountBalance({ account: accountName })
     const confirmedBalance = Number(balanceResp.content.confirmed)
-    const requiredBalance = ironToOre(IRON_TO_SEND) + fee
 
-    if (confirmedBalance < requiredBalance) {
-      this.log(`Insufficient balance: ${confirmedBalance}. Required: ${requiredBalance}`)
+    if (confirmedBalance < fee + ironToOre(minDepositSize)) {
+      this.log(
+        `Insufficient balance: ${oreToIron(
+          confirmedBalance,
+        )} IRON. At minimum, depositing requires the fee (${fee} ORE) plus a minimum deposit (${minDepositSize} IRON)`,
```

Comment:
Should we convert this `fee` log to IRON? Seemed like it would be a bit extreme to put in IRON (would be like 0.0000000001 IRON, right?)

Reply:
I think so, yes. IRON at the edges, ore internally.

On a side note, I think the language could be reduced down a bit. Most of our user's do not speak english as a first language.

Reply:
Good thinking, I didn't even consider that. I will change